### PR TITLE
[oracle] Add price feeder sidecar

### PR DESCRIPTION
**Pull request by udpatil**
## Describe your changes and provide context
This is largely forked from the Kujira Oracle Price Feeder sidecar, with changes made for compatibility with sei-chain. This will need to be further updated to update the provider integrations, and to improve the documentation for running this oracle process.

Specific Files of interest:
oracle/price_feeder/oracle.go
oracle/cmd/price_feeder.go

Most of the files have minor modifications specifically for import differences and compatibility differences as opposed to complete rewrites.

## Testing performed to validate your change
Tested on local chain with some denoms.

### CODE REVIEW

File: oracle/price-feeder/oracle/oracle.go [hunk 1]
```diff
@@ -0,0 +1,581 @@
...
+		providerCandles,
+		providerPrices,
+		o.providerPairs,
+		o.deviations,
+	)
+	if err != nil {
+		return err
+	}
+
+	if len(computedPrices) != len(requiredRates) {
```
Comment (philipsu522) [inline oracle/price-feeder/oracle/oracle.go:258]: this logic seems like the price feeder is all-or-nothing. do we want to allow validators to submit prices for whatever they can get?

File: oracle/price-feeder/dockerfile/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+keyring-backend = "file"
+output = "text"
+node = "tcp://rpc-kujira.synergynodes.com:80"
+broadcast-mode = "sync"
+EOF
+```
+
+## Recover oracle `keyring-file` to local file
+```bash
+kujirad keys add oracle --keyring-backend file --recover
```
Comment (philipsu522) [inline oracle/price-feeder/dockerfile/README.md]: should we change this (and all other references to kujira) to sei?

File: oracle/price-feeder/oracle/oracle.go (hunk 1 above)
Comment (udpatil) [reply oracle/price-feeder/oracle/oracle.go:258]: we could if we want, but I'd prefer to first add this as a TODO and get the sidecar merged in. We can already tune the config toml to exclude certain denominations and thats no issue even if we dont support all whitelisted assets, whereas this relates to the presence of rates for the denoms we have in the config. If it sounds good I'll add in a TODO for more graceful exclusion of prices that are missing

File: oracle/price-feeder/dockerfile/README.md (hunk 2 above)
Comment (udpatil) [reply oracle/price-feeder/dockerfile/README.md]: yup good call

File: oracle/price-feeder/oracle/oracle.go (hunk 1 above)
Comment (udpatil) [reply oracle/price-feeder/oracle/oracle.go:258]: added todo

**Review by philipsu522 — APPROVED** [review]

**Pull request merged**